src-ui/src/app/components/document-detail/document-detail.component.ts

```diff
@@ -48,8 +48,17 @@ export class DocumentDetailComponent implements OnInit {
     tags: new FormControl([])
   })
 
+  @ViewChild('nav') nav: NgbNav
+  @ViewChild('pdfPreview') set pdfPreview(element) {
+    // this gets called when compontent added or removed from DOM
+    if (element && element.nativeElement.offsetParent !== null) { // its visible
+
+      setTimeout(()=> this.nav?.select(1));
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592509)
```suggestion
      setTimeout(() => this.nav?.select(1));
```